svf-llvm/lib/LLVMModule.cpp

```diff
@@ -182,27 +187,40 @@ void LLVMModuleSet::build()
     if (Options::SVFMain())
         addSVFMain();
 
+    CallGraphBuilder callGraphBuilder;
+    callgraph = callGraphBuilder.createSVFIRCallGraph(svfModule);
+
     createSVFDataStructure();
     initSVFFunction();
 
+    // iterate over CallGraphNode2SVFFunMap
+    for (auto& item : CallGraphNode2SVFFunMap)
+    {
+        CallGraphNode* callNode = const_cast<CallGraphNode*>(item.first);
+        SVFFunction* fun = item.second;
+        callNode->init(fun->getFunctionType(),
```

Comment:
Too many fields in the classs. Need to simplify fields and use methods for flags and some fields